site/content/docs/quickstart/_index.md

```diff
@@ -0,0 +1,509 @@
+        "repairEnabled": false,
+        "retentionOptions": {
+          "retentionPeriodNanos": "43200000000000",
+          "blockSizeNanos": "1800000000000",
+          "bufferFutureNanos": "120000000000",
+          "bufferPastNanos": "600000000000",
+          "blockDataExpiry": true,
+          "blockDataExpiryAfterNotAccessPeriodNanos": "300000000000",
+          "futureRetentionPeriodNanos": "0"
+        },
+        "snapshotEnabled": true,
+        "indexOptions": {
+          "enabled": true,
+          "blockSizeNanos": "1800000000000"
+        },
+        "schemaOptions": null,
+        "coldWritesEnabled": false,
+        "runtimeOptions": null
+      }
+    }
+  }
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## Writing and Querying Metrics
+
+### Writing Metrics
+
+M3 supports ingesting [statsd](https://github.com/statsd/statsd#usage) and [Prometheus](https://prometheus.io/docs/concepts/data_model/) formatted metrics.
+
+This quickstart focuses on Prometheus metrics which consist of a value, a timestamp, and tags to bring context and meaning to the metric.
+
+You can write metrics using one of two endpoints:
+
+-   _[{{% apiendpoint %}}prom/remote/write](/docs/m3coordinator/api/remote/)_ - Write a Prometheus remote write query to M3DB with a binary snappy compressed Prometheus WriteRequest protobuf message.
+-   _{{% apiendpoint %}}json/write_ - Write a JSON payload of metrics data. This endpoint is quick for testing purposes but is not as performant for production usage.
+
+For this quickstart, use the _{{% apiendpoint %}}json/write_ endpoint to write a tagged metric to M3DB with the following data in the request body, all fields are required:
+
+-   `tags`: An object of at least one `name`/`value` pairs
+-   `timestamp`: The UNIX timestamp for the data
+-   `value`: The value for the data, can be of any type
+
+{{% notice tip %}}
+The examples below use `__name__` as the name for one of the tags, which is a Prometheus reserved tag that allows you to query metrics using the value of the tag to filter results.
+{{% /notice %}}
+
+{{% notice tip %}}
+Label names may contain ASCII letters, numbers, underscores, and Unicode characters. They must match the regex `[a-zA-Z_][a-zA-Z0-9_]*`. Label names beginning with `__` are reserved for internal use. [Read more in the Prometheus documentation](https://prometheus.io/docs/concepts/data_model/#metric-names-and-labels).
+{{% /notice %}}
+
+{{< tabs name="write_metrics" >}}
+{{< tab name="Command 1" >}}
+
+{{% codeinclude file="quickstart/write-metrics-1.sh" language="shell" %}}
+
+{{< /tab >}}
+{{< tab name="Command 2" >}}
+
+{{% codeinclude file="quickstart/write-metrics-2.sh" language="shell" %}}
+
+{{< /tab >}}
+{{< tab name="Command 3" >}}
+
+{{% codeinclude file="quickstart/write-metrics-3.sh" language="shell" %}}
+
+{{< /tab >}}
+{{< /tabs >}}
+
+### Querying metrics
+
+M3DB supports three query engines: Prometheus (default), Graphite, and the M3 Query Engine.
```

Comment:
Do we want to mention the m3 query engine? @robskillington

Reply:
We probably should I guess? It was mentioned in the old doc, I removed all the mentions of "missing features" etc.